.husky/pre-push

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
```

Comment:
**question**: should I add `npm run lint` as well here?

Reply:
If it takes more than 5 seconds I would say no

Reply:
It took `6.6262s` on my machine with m1 😄